tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +46,71 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownUserAndGroup() throws IOException, AlluxioException, Exception {
+    clearLoginUser();
+    GroupMappingServiceTestUtils.resetCache();
+    Configuration.set(PropertyKey.SECURITY_GROUP_MAPPING_CLASS,
+        IdentityUserGroupsMapping.class.getName());
+    Configuration.set(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName());
+    Configuration.set(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true");
+
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    String nonexistUser = "nonexistuser";
+    List<String> groups = CommonUtils.getGroups(newOwner);
+    String nonexistGroup = "nonexistgroup";
+    Random rand = new Random();
+    int numOfGroups = groups.size();
+    // chown legalUser:legalGroup
+    for (int i = 0; i < 8; i++) {
+      try {
+        int k = rand.nextInt(numOfGroups);
+        String group = groups.get(k);
+        mFsShell.run("chown", newOwner + ":" + group, "/testFile");
+        String currentGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+        Assert.assertEquals(group, currentGroup);
+      } catch (IllegalArgumentException e) {
+        mOldOutput.format("The user %s does not belong any group.%n", newOwner);
```

Comment:
should this case with `legalUser:legalGroup` succeed? For those cases you might not want to catch the exception... it the test failed for some reason, this test method should throw the exception and fail.